src/Raven.Server/RavenServer.cs

```diff
@@ -815,10 +815,24 @@ private void SendErrorIfPossible(TcpConnectionOptions tcp, Exception e)
             return stream;
         }
 
+
+
         private bool TryAuthorize(RavenConfiguration configuration, Stream stream, TcpConnectionHeaderMessage header, out string msg)
```

Comment:
The error we'll get is an authorization exception, no? That is pretty misleading.